src/routes/autorag.ts

```diff
@@ -401,3 +447,230 @@ export async function handleAutoRAGJobs(c: ContextWithAuth) {
     );
   }
 }
+
+/**
+ * Check the status of a single AutoRAG job and update file status accordingly
+ */
+async function checkSingleJobStatus(
+  job: any,
+  env: any
+): Promise<{ status: string; updated: boolean }> {
+  const fileDAO = new FileDAO(env.DB);
+
+  // Call the Cloudflare AutoRAG API to check job status
+  const accountId = env.AUTORAG_ACCOUNT_ID;
+  const apiUrl = env.AUTORAG_API_URL;
+
+  if (!accountId || !apiUrl) {
+    throw new Error("AutoRAG configuration missing: ACCOUNT_ID or API_URL");
+  }
+
+  const jobDetailsUrl = API_CONFIG.ENDPOINTS.AUTORAG.API.JOB_DETAILS(
+    accountId,
+    job.rag_id,
```

Comment:
minor: inconsistent naming (underscore v camelcase)

Reply:
file_name is like that too. I'm going to run another clean slate and get us consistent here